compiler/src/dotty/tools/dotc/core/tasty/TastyPrinter.scala

```diff
@@ -22,8 +23,8 @@ class TastyPrinter(bytes: Array[Byte])(implicit ctx: Context) {
 
   def printNames() =
     for ((name, idx) <- nameAtRef.contents.zipWithIndex) {
-      val index = "%4d: ".format(idx)
-      println(index + nameToString(name))
+      val index = Magenta("%4d".format(idx))
```

Comment:
I would add a layer of abstraction, e.g. "NameColor" instead of "Magenta"

Reply:
Will do.